src/platform/forms/tests/fixtures/mocks/mockXX123Get.js

```diff
@@ -0,0 +1,23 @@
+import moment from 'moment';
+/* eslint-disable camelcase */
+
+const mockXX123Get = {
+  formData: {
+    veteranFullName: {
+      first: 'Micky',
+      last: 'Mouse',
+    },
+    email: 'test@test.com',
```

Comment:
Where the previous tests may have been checking the prefilled SSN value, we may be able to check for an email prefill while still leaving the name fields open for a `.fill()`.

Reply:
I have updated the suite to add back in the tests that look for previously saved data to be prefilled and am using the email field to test